ovos_stt_http_server/__init__.py

```
@@ -131,8 +122,10 @@ def stats(request: Request):
     @app.post("/stt", response_class=PlainTextResponse)
     async def get_stt(request: Request):
         lang = str(request.query_params.get("lang", Configuration().get("lang", "auto"))).lower()
+        sr = int(request.query_params.get("sample_rate", 16000))
+        sw = int(request.query_params.get("sample_width", 2))
         audio_bytes = await request.body()
-        audio = bytes2audiodata(audio_bytes)
+        audio = AudioData(audio_bytes, sr, sw)
         if lang == "auto":
             lang, prob = model.detect_language(audio_bytes)
```

Ruff (0.14.10)

Unpacked variable `prob` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)